holoviews/plotting/util.py

```diff
@@ -451,6 +453,31 @@ def get_min_distance(element):
     return 0
 
 
+class univariate_composite(Operation):
```

Comment:
Minor point, I think you could call this ``univariate_compositor``.

Reply:
Agreed.